app/controllers/idv/in_person/verify_info_controller.rb

```diff
@@ -46,6 +46,7 @@ def self.step_info
             idv_session.resolution_successful = nil
             idv_session.verify_info_step_document_capture_session_uuid = nil
             idv_session.threatmetrix_review_status = nil
+            idv_session.source_check_vendor = nil
```

Comment:
Do we have any specs for these undo_step things that need to be updated? Or do we just test this as a side effect of other tests

Reply:
Yep, seems like undo step validation is primarily tested through feature specs.